pkg/api/types.go

```diff
@@ -5,4 +5,8 @@ const (
 	TlsAcmePausedAnnotation             = "kubernetes.io/tls-acme-paused"
 	AcmeAwaitingAuthzUrlAnnotation      = "kubernetes.io/tls-acme-awaiting-authorization-at-url"
 	AcmeAwaitingAuthzUrlOwnerAnnotation = "kubernetes.io/tls-acme-awaiting-authorization-owner"
+	// TlsSecretPrefixAnnotation describes the annotation used to determine a
+	// possible prefix for the TLS secret name used to store the TLS certificate
+	// obtained
+	TlsSecretPrefixAnnotation = "kubernetes.io/tls-acme-secret-prefix"
```

Comment:
I'd prefer to configure the whole secret name then a prefix.

Would be nice to cover with e2e.